pkg/dns-server/resolver/utils.go

```diff
@@ -0,0 +1,31 @@
+package resolver
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/miekg/dns"
+)
+
+func (d *SimpleDNSResolver) domainFromName(name string) (string, error) {
+	split := dns.SplitDomainName(name)
+	if len(split) < 1 {
+		return "", fmt.Errorf("Wrong DNS name: %s", name)
```

Comment:
there is a convention in Go that errors should be lowercased. Additionally, we use `"github.com/pkg/errors"` across the project.